### ISSUE TITLE
[ACCESSIBILITY] Dynamic screen changes not being announced by assistive technology

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#2044]
**Describe the accessibility issue**
When a non sighted user, using a screen reader, is checking items off their to do list in Wellness, they do not hear anything upon the appearance of the "Clear checked items" button but they should, as well as if the button was "checked" or "unchecked"

**To Reproduce
Steps to reproduce the behavior:**
_Using VoiceOver, gesture through app_
Go to '.Wellness.'
Click on '..TO Do list.'
(If you have 0 items, add 3-5) 
Select the radial button to check off item from list

**Expected behavior**
First the user should have "checked" or "unchecked" announced to them based on action completed. In the event of checking an item off, the "Clear Completed items" button also appears and should receive focus and announce to user without further gesturing needed. 

**Actual behavior**
 "checked" or "unchecked" is NOT announced to user based on action completed. When user is checking an item off, the "Clear Completed items" button also appears but does not receive focus and announce to user without further gesturing needed. 

**Screenshots
If applicable, add screenshots to help explain your problem.**
1. With one item checked, the "Clear Completed Items" button appears
![20220718_104303](https://user-images.githubusercontent.com/82892873/179615582-f49ba521-7c2b-473d-84a0-54de3f37f40e.jpg)

**Smartphone (please complete the following information):**
iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Additional context**

**Comment by acovher2** [comment]
Found in 4.0.53

**Comment by TodorBachvarov** [comment]
Checkbox pronouncement fixed in version 4.1.1
